.jenkins/Jenkinsfile

```diff
@@ -33,7 +33,7 @@ def ACCTest(String label, String version, String compiler, String build_type) {
                         ninja -v
                         LD_LIBRARY_PATH=${WORKSPACE}/src/Linux ctest --output-on-failure
                        """
-                oe.Run(compiler, task)
+                dcap.Run(compiler, task)
```

Comment:
Can dcap be a global at the top like the oe one was?

Reply:
No. The oe is using it as a global shared library and it needs a separate repo. We want to avoid the separate repo so the global share library is not working.